environment.yml

```
@@ -5,7 +5,7 @@ channels:
 dependencies:
   - python=3.9
   - gmt=6.2.0
-  - pygmt=0.4.1
+  - pygmt=0.5.0
   - jupyterlab=3.1.1
```

Maybe bump jupyterlab to [3.2.1](https://github.com/jupyterlab/jupyterlab/releases/tag/v3.2.1) too.

```suggestion
  - jupyterlab=3.2.1
```